test/boilerplate.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var path = require('path');
+var assert = require('yeoman-assert');
+var helpers = require('yeoman-generator').test;
+
+describe('node:boilerplate', function () {
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../generators/boilerplate'))
+      .on('end', done);
+  });
+
+  it('creates files', function () {
+    assert.file('lib/index.js');
+    assert.file('test/index.js');
+  });
```

Comment:
instead of having passing tests from the beginning, in the past [we decided not to do so](https://github.com/yeoman/generator-node/blob/master/app/templates/test/test.js#L8)